sdk/src/main/scala/com/horizen/account/api/http/AccountTransactionApiRoute.scala

```diff
@@ -68,28 +73,144 @@ case class AccountTransactionApiRoute(override val settings: RESTApiSettings,
-        val tmpEtherTx = new EthereumTransaction(rawTransaction)
+        // get all accounts
+        val allAccounts = wallet.secretsOfType(classOf[PrivateKeySecp256k1])
+        val secret = allAccounts.find(
+          a =>
+            sidechainNodeView.getNodeState.getBalance(a.asInstanceOf[PrivateKeySecp256k1].publicImage.address)
+              .getOrElse(BigInteger.valueOf(0)).compareTo(valueInWei) >= 0// TODO account for gas
+        )
+        secret match {
+          case Some(secret) =>
+            val accountSecret = secret.asInstanceOf[PrivateKeySecp256k1]
+            val nonce = BigInteger.valueOf(
+              sidechainNodeView.getNodeState.getAccount(accountSecret.publicImage.address).nonce)
+
+            val destAddress = body.toAddress
+            val gasPrice = BigInteger.valueOf(1) // TODO actual gas implementation
+            val gasLimit = BigInteger.valueOf(1) // TODO actual gas implementation
+            val tmpTx = new EthereumTransaction(
+              destAddress,
+              nonce,
+              gasPrice,
+              gasLimit,
+              valueInWei,
+              "",
+              null
+            )
+            val messageToSign = tmpTx.messageToSign()
+            val msgSignature = accountSecret.sign(messageToSign)
+            val ethereumTransaction = new EthereumTransaction(
+              new SignedRawTransaction(
+                tmpTx.getTransaction.getTransaction,
+                new SignatureData(msgSignature.getV, msgSignature.getR, msgSignature.getV)
+              )
+            )
+            validateAndSendTransaction(ethereumTransaction)
+          case None =>
+            ApiResponseUtil.toResponse(ErrorInsufficientBalance("ErrorInsufficientBalance", JOptional.empty()))
+        }
+      }
+    }
+  }
+
+  /**
+   * Create and sign a core transaction, specifying regular outputs and fee. Search for and spend proper amount of regular coins. Then validate and send the transaction.
+   * Return the new transaction as a hex string if format = false, otherwise its JSON representation.
+   */
+  def createEIP1559Transaction: Route = (post & path("createEIP1559Transaction")) {
+    entity(as[ReqCreateEIP1559Transaction]) { body =>
+      val ethTx = new EthereumTransaction(
+        body.payload.chainId,
+        body.payload.to.orNull,
+        body.payload.nonce,
+        body.payload.gasLimit,
+        body.payload.maxPriorityFeePerGas,
+        body.payload.maxFeePerGas,
+        body.payload.value,
+        body.payload.data,
+        if (body.payload.signature_v.isDefined)
+          new SignatureData(
+            body.payload.signature_v.get,
+            body.payload.signature_r.get,
+            body.payload.signature_s.get)
+        else
+          null
+      )
+      // lock the view and try to create CoreTransaction
+      applyOnNodeView { sidechainNodeView =>
+        validateAndSendTransaction(ethTx)
```

Comment:
I think, in case tx was not yet signed, we need to find the proper account and sign tx, otherwise `validateAndSendTransaction` will always return an error, because tx has no valid signature.
We need to do something like in `sendCoinsToAddress` method.

Same in `createLegacyTransaction`